website/docs/d/projectLogging.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "rancher2"
+page_title: "Rancher2: rancher2_project_logging"
+sidebar_current: "docs-rancher2-resource-cluster_logging"
+description: |-
+  Get information on a Rancher v2 Project Logging resource. 
+---
+
+# rancher2\_project\_logging
+
+Use this data source to retrieve information about a Rancher v2 Project Logging.
+
+## Example Usage
+
+```hcl
+data "rancher2_project_logging" "foo" {
+  name = "foo"
+  project_id = "<project_id>"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project_id` - (Required) The project id to configure logging (string)
+* `name` - (Required) The name of the cluster logging config (string)
```

Comment:
There is on only one per project, and cluster. Why do they need the name? Type maybe...

Reply:
You are right. Addressing it to let name as computed field.

Reply:
Fixed `name` argument as computed.